app/views/result/index.html.erb

```diff
@@ -4,6 +4,13 @@
 
 <div class="wrapper">
   <a name="top"></a>
+  <button class="button show-hide-search" id="show-hide-search">Show Search</button>
+
+  <div class="new-search" id="new-search">
+    <h2>New Search</h2>
```

Comment:
Is this a `new search` or is this the search results?

Reply:
this is the container for the new search box

Reply:
I know it shows the search details, I am assuming it is prepopulated with the existing search values? if so it isn't a new search, it would be more of a refinement.. 

and if we are populating the values, why not, as that would mean a typo would result in the user needing to re-enter all query values...

Reply:
point taken, pre-populating should be easy enough :)